docs/pct/README.md

```diff
@@ -0,0 +1,291 @@
+* [Overriding Template Defaults](#overriding-template-defaults)
+
+## Overview
+
+`Puppet Content Templates` (PCT) allow you to quickly create Puppet content from scaffolding templates built by Puppet and the Community.
+
+> :warning: PCT is currently in an EXPERIMENTAL phase and feedback is encouraged via [pdk/discussions](https://github.com/puppetlabs/pdk/discussions) and starting a `feedback` post.
+
+## Getting Started
+
+Install the `exprimental` PCT release available from [github.com/puppetlabs/pdkgo/releases/](https://github.com/puppetlabs/pdkgo/releases/)
+
+Templates currently come in 2 flavours: `project` and `item`.
+
+* A `project` is a template containing many files in a particular structure. _These are great starting points._
+* An `item` is a template that will supplement a project or existing content.
+
+### pct new
+
+PCT is available through the `pct new` command.
+
+The `--list` or `-l` flag displays a list of locally available templates located in the default template location `$HOME/.pdk/pct/`. The list of templates is also available by calling `pct new` without flags.
+
+``` bash
+pct new
+pct new --list
+```
+
+Example output:
+
+```bash
+           DISPLAYNAME          |          NAME           |  TYPE
+--------------------------------+-------------------------+----------
+                                |                         |
+  Bolt Plan                     | bolt-plan               | item
+  Bolt Project                  | bolt-project            | project
+  Bolt PowerShell Task          | bolt-pwsh-task          | item
+  Bolt YAML Plan                | bolt-yaml-plan          | item
+  Puppet Module Managed Gemfile | git-attributes          | item
+  Puppet Class                  | puppet-class            | item
+  Puppet Content Template       | puppet-content-template | project
+  Puppet Defined Type           | puppet-defined-type     | item
+  Puppet Fact                   | puppet-fact             | item
+  Puppet Module                 | puppet-module           | project
+  Puppet Resource API Provider  | rsapi-provider          | item
+  Puppet Resource API Transport | puppet-transport        | item
+```
+
+Using the available templates above, its time to generate some content.
+
+``` bash
+pct new <template>
+```
+
+Replace `<template>` with the `name` of the template containing the content you want.
+
+By default the `new <template>` function will use the directory name of your current working directory to "name" your new content.
+To override this behavour use the `--name` or `-n` flag.
+
+``` bash
+pct new <template> --name MyProject
+```
+
+By default the `new <template>` function will output the template content to the current working directory.
+To override this behavour use the `--output` or `-o` flag.
+
+``` bash
+pct new <template> --output /path/to/your/project
+```
+
+> :memo: Not all templates require a `name` and providing one will have no effect on the generated content.
+
+Example usage:
```

Comment:
```suggestion
### Example workflow
```

I think this deserves it's own sub section with some verbage about how the building block approach can stitch together a set of content into a final product